samples/vmc/sddc/deploy_ovf_template_cloud.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+"""
+* *******************************************************
+* Copyright VMware, Inc. 2018. All Rights Reserved.
+* SPDX-License-Identifier: MIT
+* *******************************************************
+*
+* DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+* EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+* WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+* NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+"""
+
+__author__ = 'VMware, Inc.'
+__vcenter_version__ = 'VMware Cloud on AWS'
+
+from com.vmware.content.library_client import Item
+from com.vmware.vcenter.ovf_client import LibraryItem
+from com.vmware.vcenter_client import ResourcePool
+from vmware.vapi.vsphere.client import create_vsphere_client
+
+from samples.vsphere.common import sample_cli, sample_util
+from samples.vsphere.common.id_generator import generate_random_uuid
+from samples.vsphere.contentlibrary.lib.cls_api_helper import ClsApiHelper
+
+
+class DeployOvfTemplateCloud:
```

Comment:
nit: Do we need to suffix the class and module name with 'cloud'? Since this is located in the vmc/sddc package, cloud should be implied. I understand that it is to distinguish between the old one in vsphere, so I'm okay to leave as is.

Reply:
removed.